src/lib.rs

```diff
@@ -74,7 +74,7 @@ pub fn parse_file(path: impl AsRef<Path>) -> Result<Expr, Error> {
 }
 
 #[cfg(test)]
-use {assert_json_diff as _, rstest as _};
+use {::duckdb as _, assert_json_diff as _, rstest as _};
```

Comment:
This import became required in `rustc` v1.91 ... we use the crate in an integration test, but I guess v1.91 doesn't pick that up?